extism_pdk.go

```diff
@@ -92,7 +92,9 @@ func OutputJSON(v any) error {
 		return err
 	}
 
-	OutputMemory(AllocateBytes(b))
+	mem := AllocateBytes(b)
+	defer mem.Free()
+	OutputMemory(mem)
```

Comment:
If we'd retain memory that needs to be accessed by the host elsewhere, shouldn't that also be true here?

Reply:
So my thought here was that the host would output the information immediately and the memory could be freed. That seemed to work in the MoonBit PDK but this is one reason why I think a suite of conformance tests would be fantastic... To help answer questions like this one.